samples/ImplicitFlow/AureliaOidcClientJs/aurelia_project/aurelia.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
90% of this is the out-of-the-box aurelia-cli boilerplate for a TypeScript project.